app/components/post_with_channel_info/index.ts

```diff
@@ -3,16 +3,24 @@
 
 import {withDatabase} from '@nozbe/watermelondb/DatabaseProvider';
 import withObservables from '@nozbe/with-observables';
+import {of as of$} from 'rxjs';
 
+import {observePostSaved} from '@queries/servers/post';
 import {observeIsCRTEnabled} from '@queries/servers/thread';
 
 import PostWithChannelInfo from './post_with_channel_info';
 
 import type {WithDatabaseArgs} from '@typings/database/database';
+import type PostModel from '@typings/database/models/servers/post';
 
-const enhance = withObservables([], ({database}: WithDatabaseArgs) => {
+type OwnProps = {
+    post: PostModel;
+    skipSavedPostsHighlight?: boolean;
+} & WithDatabaseArgs;
+const enhance = withObservables(['post', 'skipSavedPostsHighlight'], ({database, post, skipSavedPostsHighlight}: OwnProps) => {
```

Comment:
why you no put space here :D



![Alt Text](https://i.giphy.com/media/xL7PDV9frcudO/giphy.webp)

Reply:
Because that is one more byte going through the wire! Don't you think about the trees and natures? :P

(Will fix)